src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,29 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  modalTodo: Todo;
+  setModalTodo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ modalTodo, setModalTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    getUser(modalTodo.userId)
+      .then(setUser)
+      .finally(() => setIsLoading(true));
```

Comment:
That is strange, you should set it to true when you start loading, and then remove it after it is finished.